src/pages/starter-kit/checkout/eds.md

```diff
@@ -8,127 +8,21 @@ keywords:
 
 # Integrate with EDS storefront
 
-Integrate your Out-of-Process Payment Extensions (OOPE) payment method with the Edge Delivery Service (EDS) Storefront.
+Integrate your Out-of-Process Extensions (OOPE) with the Edge Delivery Service (EDS) Storefront.
 
 ## Prerequisites
 
-The following prerequisites are required to integrate your OOPE payment method with the EDS Storefront:
+The following prerequisites are required to integrate your OOPE integration with the EDS Storefront:
```

Comment:
```suggestion
The following prerequisites are required to enable your OOPE integration with the EDS Storefront:
```

Just trying to avoid having the words integrate and integration so close to each other in the same sentence.